approval_polls/templates/approval_polls/my_polls.html

```diff
@@ -10,16 +10,50 @@
     <div class='row-fluid top-buffer'>
       <div class='col-md-12'>
         {% for poll in latest_poll_list %}
-          <div class='well well-sm'>
+          <div class='well well-sm' id='poll{{ poll.id }}'>
             <a href="{% url 'approval_polls:detail' poll.id %}">{{ poll.question }}</a>
-            <a class='pull-right btn btn-danger btn-xs'> remove </a>
+            <a class='pull-right btn btn-danger btn-xs' onClick="showAlert({{ poll.id }})"> remove </a>
             <a class='pull-right btn btn-warning btn-xs right-buffer'> suspend </a>
             <a class='pull-right btn btn-primary btn-xs right-buffer'> edit </a> 
             <p>
               Published on {{ poll.pub_date }}
             </p>
+            <div class="alert alert-warning alert-dismissible " role="alert" id='alert{{ poll.id }}' style="display:none">
+              You are going to delete this poll. Are you sure?
+              <button class='pull-right ' type="button" onClick="hideAlert({{ poll.id }})">No</button>
+              <button class='pull-right right-buffer' type="button" id="delete_poll{{ poll.id }}" onClick="delete_poll({{ poll.id }})" >Yes</button>
+            </div>
 
           </div>
+          <script>
```

Comment:
Looks like you're adding a script for each poll element. A more efficient way to do this would be to have a single script that adds event handlers to all relevant divs on the page. 
It would be better to move the script to a separate file like [create.js](https://github.com/electology/approval_frame/blob/master/approval_polls/static/approval_polls/create.js) or [embed_instructions.js](https://github.com/electology/approval_frame/blob/master/approval_polls/static/approval_polls/embed_instructions.js)